.github/workflows/update-docker-requirements.yml

```diff
@@ -56,10 +69,8 @@ jobs:
           # prettier-ignore
           body:
             https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-          reviewers: ${{ github.actor }}
+          reviewers: ${{ env.pr_author }},${{ env.pr_mergedBy }}
```

Comment:
We can leave the merger here, but I don't know whether we can put an arbitrary user as reviewer. For example, I cannot ask for your review on a PR. We should just ping the author in the body.